docs/vc-http-api.yml

```diff
@@ -282,6 +282,18 @@ components:
           "domain": "example.com",
           "challenge": "d436f0c8-fbd9-4e48-bbb2-55fc5d0920a8",
         }
+    DeriveCredentialOptions:
+      type: object
+      additionalProperties: false
+      description: Options for specifying how the derived credential is created.
+      properties:
+        nonce:
+          type: string
+          description: A nonce provided by the holder of the credential to be included into the LinkedDataProof. For example 6e62f66e-67de-11eb-b490-ef3eeefa55f2
```

Comment:
If we want to be able to generate a nonce, we might need a bit more nuance in this description or another option.

Reply:
To make the default less ambiguous, we can either state that the default is to generate a nonce or that the default is empty nonce.

In the case where the default is a generated nonce, we would also want to specify the behavior of an empty nonce option being provided (use empty nonce; generate nonce; disallow).

Note: the [BBS+ Signatures spec](https://w3c-ccg.github.io/ldp-bbs2020/#nonce) currently states:
> A linked data document featuring a BBS+ proof of knowledge linked data proof MAY contain a nonce attribute.

Reply:
another example of harmful MAY... why is this not a MUST and be of min length ... etc... @tplooker can we make this a MUST ?

Reply:
https://github.com/w3c-ccg/ldp-bbs2020/pull/43